docs/shared/reference/project-configuration.md

```diff
@@ -393,8 +393,7 @@ You can also remove a dependency as follows:
 
 ### Ignoring a project
 
-Nx will add every project with a `package.json` file in it to its project graph. If you want to ignore a particular
-project, add the directory to your `.nxignore` file.
+Nx will add projects with a `package.json` file that are referenced (1) in the `workspaces` property of the root `package.json`, (2) in `pnpm-workspace.yml` or (3) in `lerna.json`. If you want to ignore a particular project that those tools reference, add the directory to your `.nxignore` file.
```

Comment:
I feel this first sentence deserves its own section. It doesn't fit in the "Ignoring a project" section. We should also mention the `nx/plugins/package-json` plugin. [Reference](https://github.com/nrwl/nx/issues/13063#issuecomment-1317345724)

Reply:
I can expand this into its own section, but I was intentionally not listing the `nx/plugins/package-json` plugin since it isn't recommended.

I'll make a separate recipe that shows how to add the plugin, but not add it in the project configuration reference.

Reply:
Projects should not be ignored via nxignore cause it will break the cache. The only things that nxignore should contains are transient files that are not included into any target.

Reply:
If you don't want your project, add !packages/myproject to workspaces property.